Kickstarter-iOS/Features/PledgePaymentMethods/Controller/PledgePaymentMethodsViewController.swift

```diff
@@ -194,8 +212,9 @@ final class PledgePaymentMethodsViewController: UIViewController {
           image: existingPaymentOption.image,
           label: existingPaymentOption.label
         )
+
         strongSelf.viewModel.inputs
-          .paymentSheetDidAdd(newCard: paymentDisplayData, setupIntent: clientSecret)
+          .paymentSheetDidAdd(newCard: paymentDisplayData, paymentMethod: selectedPaymentMethod)
```

Comment:
As a note this means that `PledgePaymentMethodsViewController` is diverging a bit from its cousin in Settings.